client/example-cook.json

```diff
@@ -0,0 +1,25 @@
+{
+    "defaults":{
+        "cluster": "dev",
+        "run": {
+            "mem": 128,
+            "cpus": 1
+        }
+    },
+    "clusters": [
+        {
+            "name": "dev",
+            "url": "http://127.0.0.1:12321/swagger-docs"
+        },
+        {
+            "name": "fake-dev0",
+            "url": "http://127.0.0.1:12322/swagger-docs",
+            "http_client": ["basic_auth", "127.0.0.1:12322", "u", "p"]
```

Comment:
Please use a map so that the values are self documenting. i.e. I assume the values in the list are [auth-type, url, user, password] but I'm not sure. Having it be:
```
{"auth" : {"type" : "basic_auth",
               "params" : {"user" : "u", "password" : "p"}}
 "url" : "127.0.0.1:12322",
```